### DISCUSSION TITLE
Limited results on public song request page – will it nevertheless search ALL enabled songs?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6139] [Q&A] [answered]
The number of results on the public song request page is currently limited to 1000 (default 100 pages of 10 ea.). (Which might not be a bad idea, in order not to overwhelm the user, and to make it more performant.)

Since I have ~150k titles, my question is: **Will it nevertheless _search_ the whole base of currently enabled songs (i.e., my general rotation, which is based on 3 weighted playlists which together contain ~36k songs)? And then _present_ only the first 1000 search hits?**

(Rolling Release #[6f36430](https://github.com/AzuraCast/AzuraCast/commit/6f364309bc79240888e46e4c7571464db95bde83) (2023-03-06 4:41))

**Comment by BusterNeece** [comment, marked as answer]
Yes.
